skimage/feature/tests/test_sift.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_almost_equal, assert_equal
+
+from skimage import data
+from skimage._shared.testing import test_parallel
+from skimage.feature import SIFT
+from skimage.util.dtype import _convert
+
+img = data.coins()
+
+
+@test_parallel()
+@pytest.mark.parametrize(
+    'dtype', ['float32', 'float64', 'uint8', 'uint16', 'int64']
```

Comment:
@grlee77 do we have a preference for writing `np.float32` over `'float32'`?

Reply:
I don't have a strong preference either way. I think using strings may have made the parameterized test case names clearer with older pytest versions, but I'm pretty sure for > 1.5 years now it checks the `__name__` attribute of objects and uses that.